l10n_it_pec_messages/model/mail_message.py

```diff
@@ -115,6 +142,67 @@ def _get_out_server(self, cr, uid, ids, name,
         'direction': 'in'
     }
 
+    def CheckStatus(self, cr, uid, ids, context=None):
+        mail_mail_pool = self.pool['mail.mail']
+        if context is None:
+            context = {}
+        if not hasattr(ids, '__iter__'):
+            ids = [ids]
+        error_lst = []
+        receipt_list = []
+        completed = True
+        res = {}
+        if ids:
+            for message in self.browse(cr, uid, ids, context=context):
+                mail_sent_ids = mail_mail_pool.search(
+                    cr,
+                    uid,
+                    [('mail_message_id', '=', message.id)],
+                    context=context
+                )
+                if not mail_sent_ids:
+                    _logger.exception(
```

Comment:
@Lorenzo riguardo al logging ho verificato la documentazione, in questo caso è corretto usare error. Modifico